app/src/main/java/com/duckduckgo/app/global/AppUrl.kt

```diff
@@ -36,10 +36,14 @@ class AppUrl {
         const val DEV_MODE = "test"
         const val LANGUAGE = "lg"
         const val COUNTRY = "co"
+        const val HIDE_SERP = "ko"
+        const val VERTICAL = "ia"
+        const val VERTICAL_REWRITE = "iar"
```

Comment:
read https://app.asana.com/0/1157893581871903/1175183575918486 for more info on why there are two parameters